src/Viewer/Viewer.test.ts

```diff
@@ -12,7 +12,7 @@ type UnusedProps = UnusedCesiumProps<
   typeof cesiumEventProps,
   IgnoredProps
 >;
-type IgnoredProps = "contextOptions"; // contextOptions is actually used
+type IgnoredProps = "contextOptions" | "terrainProvider"; // contextOptions is actually used
```

Comment:
is there reason to ignore terrainProvider?

Reply:
Cesium changed a lot of API as asynchronous. `createWorldTerrainAsync()` returns Promise, but Cesium's Viewer doesn't allow to pass Promise to terrainProvider props. So I fixed to extend the type of terrainProvider to take Promise. Due to that, terrainProvider is defined as otherProps.